api/src/indicators.py

```diff
@@ -287,6 +287,46 @@ def get_latest_indicators(size=10000):
         IndicatorsSchemaArray.append(res.get("_source"))
     return IndicatorsSchemaArray
 
+@router.get("/indicators/ncfiles", response_model=List[IndicatorSchema.IndicatorsSearchSchema])
+def get_nc_file_indicators(size=10000):
+    # match just .nc files
+    query = {
+        "_source": [
+            "description",
+            "name",
+            "id",
+            "created_at",
+            "deprecated",
+            "maintainer.name",
+            "maintainer.email",
+            "fileData.raw.url",
+            "fileData.raw.rawFileName",
+        ],
+        "query": {
+            "bool": {
+                "must": [
+                    {"match_all": {}},
```

Comment:
Ah, good point. Might be worth changing it to an `and` version in case a file has just one (though any file that goes through the data annotation pipeline _should_ have both, as they get set when we do the final upload steps).